src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -26,9 +29,29 @@ import UpdatePhone from './components/UpdatePhoneDialog';
 import styles from './PersonalInfoList.module.css';
 import AlumniUpdateForm from './components/AlumniUpdateForm';
 import CliftonStrengthsService from 'services/cliftonStrengths';
+import SLock from './Salsbury.png';
+import DPLock from './DandP.png';
+import DDLock from './DandD.png';
+import useWindowSize from 'hooks/useWindowSize';
 
 const PRIVATE_INFO = 'Private as requested.';
 
+// const Lock1 = width >= 200 ? : SLock;
+
+// const Lock2 = width >= 200 ? : DPLock;
+
+// const Lock3 = width >= 200 ? DDLock;
```

Comment:
We prefer to delete code instead of commenting it out unless there is a good reason to save it. Since your pull request is open and ready for review, I would recommend deleting these lines.

Reply:
Thank you for the feedback, I'll be more careful next time before submitting a PR.

Reply:
Thank you for the feedback, I'll be more careful next time before submitting a PR.